### PR TITLE
refactor/adding-nannies

### PR DESCRIPTION
**Pull request by LRydeard**
## Description

Refactor to separate childminder and nanny forms, views and templates.
Minor edit to selenium tests and an additional test migration was added.

## Todo's before PR

- [x] Rebase from develop
- [x] Unit tests passed (`make test`)
- [x] PR naming according our [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [x] PR description describes the overall goals of PR commits
- [ ] Templates been checked with relevant user-story

## Migrations

- [ ] Null fields in models specifies default value
- [x] You generated and applied migrations (`make migrate`)
- [ ] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [x] Specified reviewers (even if it's yourself)
- [x] Specified assigneses (those who'll merge)
- [x] Specified labels
- [x] Specified milestone

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [ ] Code syntax formatting checked
- [ ] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [ ] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

**Comment by Thomas-95IS** [comment]
rerun jenkins build

**Comment by Thomas-95IS** [comment]
rerun jenkins build

File: arc_application/contact_centre.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def search(request):
 
         if request.method == 'GET':
             context['form'] = SearchForm()
-            return render(request, 'search.html', context)
+            return render(request, 'childminder_templates/search.html', context)
```
Comment (kingainformed) [inline arc_application/contact_centre.py]: Can we make this a variable, so if we have to rename again it's just changing one line?

File: arc_application/templates/childminder_templates/add-previous-names.html [hunk 2]
```diff
@@ -38,7 +38,7 @@ <h2 class="form-title heading-medium">
         <input type="hidden" name="referrer" value="{{ referrer }}">
         <input type="submit" name="action" class="button" value="Confirm and continue"/>
 
-        {% include 'return-to-list.html' %}
+        {% include 'childminder_templates/return-to-list.html' %}
```
Comment (kingainformed) [inline arc_application/templates/childminder_templates/add-previous-names.html]: Can this be put in shared templates folder? Nanny will need to use this too.

**Comment by kingainformed** [comment]
rerun jenkins build

**Pull request merged**